src/core/memory.cc

```diff
@@ -1832,6 +1830,23 @@ void configure_minimal() {
     init_cpu_mem();
 }
 
+static long mbind(void *addr,
+                  unsigned long len,
+                  int mode,
+                  const unsigned long *nodemask,
+                  unsigned long maxnode,
+                  unsigned flags) {
+  return syscall(
+    SYS_mbind,
+    reinterpret_cast<long>(addr),
+    len,
+    mode,
+    reinterpret_cast<long>(nodemask),
+    maxnode,
+    flags
+  );
+}
```

Comment:
Our coding style uses 4-space indents, not two.

I don't think the reinterpret_casts are required.

Reply:
Corrected. Thank you!